KomaMRIIO/Project.toml

```diff
@@ -0,0 +1,29 @@
+name = "KomaMRIIO"
+uuid = "fcf631a6-1c7e-4e88-9e64-b8888386d9dc"
+authors = ["Carlos Castillo Passi <cncastillo@uc.cl>"]
+version = "0.7.7"
+
+[deps]
+FileIO = "5789e2e9-d7fb-5bc7-8068-2c6fae9b9549"
+HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
+MAT = "23992714-dd62-5051-b70f-ba57cb901cac"
+MRIBase = "f7771a9a-6e57-4e71-863b-6e4b6a2f17df"
+MRIFiles = "5a6f062f-bf45-497d-b654-ad17aae2a530"
```

Comment:
Do you need both?

Reply:
We utilize EncodingCounters() from MRIBase and Limits() from MRIFiles.